buf.gen.yaml

```diff
@@ -18,3 +20,17 @@ plugins:
     out: pkg/proto
     opt:
       - paths=source_relative
+  - plugin: buf.build/connectrpc/swift
+    opt:
+      - GenerateAsyncMethods=true
+      - GenerateCallbackMethods=true
+      - Visibility=Public
+    out: swift
+  - plugin: buf.build/apple/swift
+    opt:
+      - Visibility=Public
+    out: swift
+  - plugin: buf.build/protocolbuffers/java
+    out: kotlin
+  - plugin: buf.build/connectrpc/kotlin
+    out: kotlin
```

Comment:
This is the only real change the rest is just generated code or readme update. Most of these files were updated from this command `./dev/gen-proto` I can cut down on the amount of touched files if I just run `buf generate` but wanted to follow the readme instructions.